run/markdown-preview/renderer/main.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const express = require('express');
+const MarkdownIt = require('markdown-it');
+
+const app = express();
+app.use(express.json());
+
+app.post('/', (req, res) => {
+  let markdown;
+  if (!req.body.markdown.data) {
```

Comment:
`.markdown` is an extra step because you are bundling as json. @grayside does the markdown need to be sent around as JSON?